plugin/tick.vim

```diff
@@ -2,7 +2,140 @@
 "
 " Much of this plugin was copied from Fatih's wonderful https://github.com/fatih/vim-go/, so send him your thanks!
 
+" install necessary Go tools
+if exists("g:tick_loaded_install")
+    finish
+endif
+let g:tick_loaded_install = 1
+
+
+" these packages are used by vim-tick and can be automatically installed if
+" needed by the user with TickInstallBinaries
+let s:packages = [
+            \ "github.com/influxdata/kapacitor/tick/cmd/tickfmt"
+            \ ]
+
+" These commands are available on any filetypes
 command! TickFmt call tick#fmt#Format()
+command! TickInstallBinaries call s:TickInstallBinaries(-1)
+command! TickUpdateBinaries call s:TickInstallBinaries(1)
+command! -nargs=? -complete=dir GoPath call go#path#GoPath(<f-args>)
```

Comment:
I'm in the same boat you are with VIMscript. I guess it's the blind leading the blind. ;) I'll look through and do a bit more testing with vim-go not installed and see if I can make there is a clean workflow.